### ISSUE TITLE
How do you use Rit in the wild?

### ISSUE THREAD

**Issue by branch14** [briandoll/Rit#5]
This is rather a list of questions than an issue.

What kind of format do you use for the content? HTML, or some other markup? Do you render it on the Rit's side or on the side of the consuming up?

How do you feel about a dropdown in the edition's form, to select one among several templating engines/renderers? Is that what you had in mind, when you were mentioning some plugin architecture?

Maybe tilt would be a good thing to have a look at here: http://github.com/rtomayko/tilt

**Comment by briandoll** [comment]
When we first built Rit, we just needed simple HTML.  As you're noticing, it would be nice to use other formats of data as well.  We have some ideas on a plugin architecture to support other formats, but haven't implemented anything yet.

I wrote up one idea for this:
http://github.com/briandoll/Rit/wiki/Plugin-Architecture

And we'd like to continue that discussion here:
http://getsatisfaction.com/rit/topics/how_should_rit_handle_rich_content

Would love your thoughts and feedback.
